Sprint-3/todo-list/script.js

```diff
@@ -1,25 +1,84 @@
 function populateTodoList(todos) {
-  let list = document.getElementById("todo-list");
-  // Write your code to create todo list elements with completed and delete buttons here, all todos should display inside the "todo-list" element.
+  const list = document.getElementById("todo-list");
+  list.innerHTML = ""; // Clear any existing todos in the HTML.
+
+  todos.forEach((todo) => {
+    const listItem = document.createElement("li");
+    listItem.className = "list-group-item d-flex justify-content-between align-items-center";
+    listItem.innerText = todo.task;
+
+    // Add strike-through if task is already completed
```

Comment:
You do not need to have comments for everything as long as your variable names are doing what it says is doing